hibernate-core/src/test/java/org/hibernate/test/batchfetch/Model.java

```diff
@@ -18,7 +18,7 @@
 	private String description;
 	private ProductLine productLine;
 	
-	Model() {}
+	public Model() {}
```

Comment:
@sebersole, this is my main point of confusion.  The non-public ctor caused the following Exception with the new CL setup in `JavassistProxyFactory`.  There are several other failures in hibernate-core for similar reasons.

`Caused by: java.lang.IllegalAccessError: tried to access method org.hibernate.test.batchfetch.Model.<init>()V from class org.hibernate.test.batchfetch.Model_$$_jvst960_2
	at org.hibernate.test.batchfetch.Model_$$_jvst960_2.<init>(Model_$$_jvst960_2.java)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at java.lang.Class.newInstance(Class.java:442)
	at org.hibernate.proxy.pojo.javassist.JavassistProxyFactory.getProxy(JavassistProxyFactory.java:136)`

Super confused why that only came up now, and not before...